source/gui/settingsDialogs.py

```diff
@@ -1719,13 +1745,13 @@ def isValid(self) -> bool:
 		enabledSpeechModes = self.speechModesList.CheckedItems
 		if len(enabledSpeechModes) < 2:
 			log.debugWarning("Too few speech modes enabled.")
-			gui.messageBox(
+			self._validationErrorMessageBox(
 				# Translators: Message shown when not enough speech modes are enabled.
-				_("At least two speech modes have to be checked."),
-				# Translators: The title of the message box
-				_("Error"),
-				wx.OK | wx.ICON_ERROR,
-				self,
+				message=_("At least two speech modes have to be checked."),
+				# Translators: Same as the label for the list of checkboxes where user can select speech modes that will
+				# be available in Speech Settings, but without keyboard accelerator (& character) nor final colon.
+				option=_("Modes available in the Cycle speech mode command"),

```

Comment:
Can't you just retrieve the label from the control and remove the unnecessary characters yourself? These two strings can easily get out of sync otherwise. I've looked at this briefly - it is likely you may need to expand `guiHelper`, so that each labeled control stores its label.

Reply:
> Can't you just retrieve the label from the control and remove the unnecessary characters yourself? These two strings can easily get out of sync otherwise. I've looked at this briefly - it is likely you may need to expand `guiHelper`, so that each labeled control stores its label.

It is not enough to just remove the "&" character from the label. E.g. in Chinese translation; see #15266 and #15258.
Then, the implementation recommended by @seanbudd in ,https://github.com/nvaccess/nvda/issues/15258#issuecomment-1667002771 was to duplicate the strings.
That's why I have followed the same approach in this PR.